package.json

```diff
@@ -13,7 +13,8 @@
   "pnpm": {
     "overrides": {
       "workerd": "^1.20240729.0",
-      "@remix-run/dev": "npm:@dario-hacking/remix-run-dev@2.9.1-vite-env-4"
+      "@remix-run/dev": "/Users/dario/Repos/my-repos/remix/packages/remix-dev",
```

Comment:
TODO: I need to do another release of our remix for and link to that before merging

(but before I can release that we'd need to release the provider packages with the changes present in this PR and use those in the remix fork)

Reply:
I've released the updated version of my remix fork and as suggested by @jamesopstad I've removed the requirement for the environment providers 👍